CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contribution Guide for PolyMath
+
+This file is currently not complete but will be improve step by step.
+
+# Release management
+
+This project use semantic versionning to define the releases. This mean that each stable release of the project will get associate a version number of the form `vX.Y.Z`. 
+
+- **X** define the major version number
+- **Y** define the minor version number 
+- **Z** define the patch version number
+
+When a release contains only bug fixes, the patch number increase. When the release contains new features backward compatibles, the minor version increase. When the release contains breaking changes, the major version increase. 
```

Comment:
We might bullet point these like:

- When a release only contains bug fixes, the patch number is incremented;
- When the release contains backwards-compatible features, the minor version is incremented;
- When the release contains breaking changes, the major version is incremented